docs/metric-update-procedure/README.md

```diff
@@ -69,10 +69,26 @@ To update the latest graphs and values for development teams, import each report
 - For `Import location`, select `Replace current sheet`, for `Separator type,` select
   `Detect automatically`, leave the `Convert text to numbers, dates, and formulas` box
    checked, and click `Import data.` The Google Sheet should now reflect the latest data.
-  
-3.3  If the report being imported is a results or coverage report. Convert the type of the date
+- If the report being imported is a results or coverage report. Convert the type of the date
      column to 'plain text' so that the graphs display at an even cadence. 
-- Highlight the `Date` column and in the top menu par select `Format > Number > Plain text`
+  - Highlight the `Date` column and in the top menu select `Format > Number > Plain text`
+
+3.3 Update Trend Table Dates
+
+- At the beginning of each week, in the Weekly Trends table:
+  - Update the `End` date to the current monday date
+  - Update the `Start` date to the last monday date
+  - Increment the week number
+  - **Example:**
+    - For week 41 in 2024, the `Start` value is 2024-09-30 and the `End` value is 2024-10-07
```

Comment:
Would that be Week 40 or 41? Here's my handy corporate Google Calendar:
![Screenshot 2024-10-09 at 2 11 44 PM](https://github.com/user-attachments/assets/3392b77d-10ed-4fe3-8606-93f2793ab500)

Reply:
Oh you are right! Week 40!... fixing all the sheets and instructions now!